src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothSocket.cs

```diff
@@ -232,6 +233,62 @@ void IBluetoothClientSocket.Disconnect()
             }
         }
 
+        /// <summary>
+        /// Registers a callback function that will be invoked when the SCO(Synchronous Connection Oriented link) state is changed
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<SocketConnectionRequestedEventArgs> ConnectionRequested
+        {
+            add
+            {
+                if (_connectionRequested == null)
+                {
+                    RegisterConnectionRequestedEvent();
+                }
+                _connectionRequested += value;
+            }
+            remove
+            {
+                _connectionRequested -= value;
+                if (_connectionRequested == null)
+                {
+                    UnregisterConnectionRequestedEvent();
+                }
+            }
+        }
+
+        private void RegisterConnectionRequestedEvent()
+        {
+            Interop.Bluetooth.SocketConnectionRequestedCallback _connectionRequestedCallback = (int socket_fd, string remoteAddress, IntPtr userData) =>
+            {
+                Log.Info(Globals.LogTag, "SocketConnectionRequestedCallback is called");
+                if (_connectionRequested != null)
+                {
+                    connectedSocket = socket_fd;
+                    GCHandle handle2 = (GCHandle)userData;
+                    _connectionRequested(handle2, new SocketConnectionRequestedEventArgs(socket_fd, remoteAddress));
+                }
+            };
+            GCHandle handle1 = GCHandle.Alloc(this);
+            IntPtr data = (IntPtr)handle1;
+            int ret = Interop.Bluetooth.SetSocketConnectionRequestedCallback(_connectionRequestedCallback, data);
```

Comment:
It seem to a global function on Core API (there is no context to handle a target, and only one callback can be set).
It can be issue when a BluetoothSocket was created more than one.

Reply:
Done. Thank  you.